### ISSUE TITLE
Support courses generated in a previous version of the framework.

### ISSUE THREAD

**Issue by brian-learningpool** [adaptlearning/adapt-contrib-media#108]
If the `_accessibility` attribute is missing from the JSON files, i.e. in version 1.x courses, the media element should still work.  Add some defensive code.

This is related to this issue in the Adapt Framework project:
https://github.com/adaptlearning/adapt_framework/issues/642
